packages/example-project/component-library/package.json

```diff
@@ -33,7 +33,6 @@
     "@stencil/angular-output-target": "^0.0.5",
     "@stencil/core": "^2.9.0",
     "@stencil/react-output-target": "^0.3.1",
-    "@stencil/svelte-output-target": "^0.0.3",
```

Comment:
This wasn't in the `package-lock.json` file to begin with, hence the lack of changes there